spotify/views/layout.hbs

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <title>{{title}}</title>
```

Comment:
Si usáis una variable aquí para el título del documento hay que pasar esta variable en todas las vistas.